proto/lekko/client/v1beta1/configuration_service.proto

```diff
@@ -93,6 +93,12 @@ message GetProtoValueRequest {
 
 message GetProtoValueResponse {
   google.protobuf.Any value = 1;
+  Any value_new = 2;
+}
+
+message Any {
```

Comment:
This an internal struct, right? Maybe name it a bit more explicitly to avoid any potential naming conflicts, like `LekkoAny`.

Reply:
The message is prefixed with the package name, so anyone using this will be able to differentiate. we're also getting rid of google's Any so there should be no confusion